starfish/pipeline/filter/mean_high_pass.py

```diff
@@ -0,0 +1,94 @@
+import argparse
```

Comment:
Good idea to use pycharm optimize imports here, it will catch unused ones and optimize their order. 

Here's what it comes up with for this module: 

```python
import argparse
from functools import partial
from numbers import Number
from typing import Callable, Optional

import numpy as np
from scipy.ndimage.filters import uniform_filter
from skimage import img_as_uint

from starfish.errors import DataFormatWarning
from starfish.image import ImageStack
from ._base import FilterAlgorithmBase
```